iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewProxy.m

```diff
@@ -2838,6 +2838,31 @@ - (void)setAccessibilityHidden:(id)accessibilityHidden
   [self replaceValue:accessibilityHidden forKey:@"accessibilityHidden" notification:NO];
 }
 
+- (NSObject *)accessibilityElement
+{
+  return self.view.accessibilityElement;
+}
+
+- (NSString *)accessibilityLabel
+{
+  return [self accessibilityElement].accessibilityLabel;
```

Comment:
Use `self.accessibilityElement` here and below (like you already did in `TiUIView.m`)

Reply:
Here 'accessibilityElement' is internal function. In TiUIView.m it is defined as property because it has to be used outside as well.

Reply:
Oh damn, didn't notice. Thanks for the clarification.